src/request/transaction.ts

```diff
@@ -121,17 +121,17 @@ export default class Transaction {
       sanitized.creditCard != null &&
       sanitized.creditCard.last4digits != null
     ) {
-      sanitized.creditCard.last_4_digits = this.creditCard!.last4digits;
+      sanitized.creditCard.last_4_digits = this.creditCard?.last4digits;
```

Comment:
For these, would it be better to resolve it by replacing `this` with `sanitized`? I think that makes more sense given the other code and should eliminate the need for `?`

Reply:
Good catch. I changed the references and also simplified the conditions.